Source/Evolve/WarpXEvolve.cpp

```diff
@@ -115,7 +117,9 @@ WarpX::Evolve (int numsteps)
             FillBoundaryE(guard_cells.ng_FieldGather, guard_cells.ng_Extra);
             FillBoundaryB(guard_cells.ng_FieldGather, guard_cells.ng_Extra);
 #ifdef WARPX_MAG_LLG
-            FillBoundaryM(guard_cells.ng_FieldGather, guard_cells.ng_Extra);
+            if (WarpX::em_solver_medium == MediumForEM::Macroscopic) {
+                FillBoundaryM(guard_cells.ng_FieldGather, guard_cells.ng_Extra);
+            }
```

Comment:
should we include:
`else {
amrex::Abort("unsupported em_solver_medium for M field");
}
`

Reply:
Sure! I added it.